### PR TITLE
minor fixes in style

### PR DESCRIPTION
**Pull request by fartariatomas**
general style fixes. I mostly pressed cmd+alt+L to reformat the code, removed some empty spaces, etc.

### CODE REVIEW

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/sink/ReThinkSinkTask.scala [hunk 1]
```diff
@@ -18,26 +18,19 @@ package com.datamountaineer.streamreactor.connect.rethink.sink
...
 import java.util
 
-import com.datamountaineer.streamreactor.connect.rethink.config.ReThinkSinkConfig
-import com.datamountaineer.streamreactor.connect.utils.ProgressCounter
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.clients.consumer.OffsetAndMetadata
-import org.apache.kafka.common.TopicPartition
-import org.apache.kafka.connect.sink.{SinkRecord, SinkTask}
-
 import scala.collection.JavaConversions._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/sink/ReThinkSinkTask.scala:21]: remove wildcard

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceConnector.scala [hunk 2]
```diff
@@ -18,13 +18,6 @@ package com.datamountaineer.streamreactor.connect.rethink.source
...
 
-import com.datamountaineer.streamreactor.connect.rethink.config.{ReThinkSourceConfig, ReThinkSourceConfigConstants}
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.common.config.ConfigDef
-import org.apache.kafka.connect.connector.Task
-import org.apache.kafka.connect.source.SourceConnector
-import org.apache.kafka.connect.util.ConnectorUtils
-
 import scala.collection.JavaConversions._
 import scala.collection.JavaConverters._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceConnector.scala:22]: wildcards remove

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceReader.scala [hunk 3]
```diff
@@ -19,18 +19,6 @@ package com.datamountaineer.streamreactor.connect.rethink.source
...
-import com.datamountaineer.streamreactor.connect.rethink.ReThinkConnection
-import com.datamountaineer.streamreactor.connect.rethink.config.{ReThinkSourceConfig, ReThinkSourceConfigConstants, ReThinkSourceSettings}
-import com.datamountaineer.streamreactor.connect.rethink.source.ReThinkSourceReader.{DataRequest, StartChangeFeed, StopChangeFeed}
-import com.rethinkdb.RethinkDB
-import com.rethinkdb.net.{Connection, Cursor}
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.connect.data.SchemaBuilder
-import org.apache.kafka.connect.source.SourceRecord
-
 import scala.collection.JavaConverters._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceReader.scala:22]: and here

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceTask.scala [hunk 4]
```diff
@@ -18,16 +18,7 @@ package com.datamountaineer.streamreactor.connect.rethink.source
...
 
-import akka.actor.{ActorRef, ActorSystem}
-import com.datamountaineer.streamreactor.connect.rethink.config.ReThinkSourceConfig
-import com.datamountaineer.streamreactor.connect.rethink.source.ReThinkSourceReader.{StartChangeFeed, StopChangeFeed}
-import com.datamountaineer.streamreactor.connect.utils.ProgressCounter
-import com.rethinkdb.RethinkDB
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.connect.source.{SourceRecord, SourceTask}
-
 import scala.collection.JavaConversions._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceTask.scala:21]: and here

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceTask.scala [hunk 5]
```diff
@@ -18,16 +18,7 @@ package com.datamountaineer.streamreactor.connect.rethink.source
...
-import scala.collection.JavaConverters._
 import scala.concurrent.Await
 import scala.concurrent.duration._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceTask.scala:23]: and here

File: kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/sink/HazelCastSinkConnector.scala [hunk 6]
```diff
@@ -18,33 +18,27 @@ package com.datamountaineer.streamreactor.connect.hazelcast.sink
 
 import java.util
 
-import com.datamountaineer.streamreactor.connect.hazelcast.config.HazelCastSinkConfig
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.common.config.ConfigDef
-import org.apache.kafka.connect.connector.Task
-import org.apache.kafka.connect.sink.SinkConnector
-
 import scala.collection.JavaConversions._
```
Comment (carolinaloureiro) [inline kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/sink/HazelCastSinkConnector.scala:21]: and here

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/sink/ReThinkHelper.scala [hunk 7]
```diff
@@ -16,12 +16,6 @@
 
 package com.datamountaineer.streamreactor.connect.rethink.sink
 
-import com.datamountaineer.streamreactor.connect.rethink.config.ReThinkSinkSetting
-import com.rethinkdb.RethinkDB
-import com.rethinkdb.net.Connection
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.connect.errors.ConnectException
-
 import scala.collection.JavaConversions._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/sink/ReThinkHelper.scala:19]: and here

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/sink/ReThinkSinkConnector.scala [hunk 8]
```diff
@@ -18,13 +18,6 @@ package com.datamountaineer.streamreactor.connect.rethink.sink
...
 import java.util
 
-import com.datamountaineer.streamreactor.connect.rethink.config.{ReThinkSinkConfig, ReThinkSinkConfigConstants, ReThinkSinkSettings}
-import com.rethinkdb.RethinkDB
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.common.config.ConfigDef
-import org.apache.kafka.connect.connector.Task
-import org.apache.kafka.connect.sink.SinkConnector
-
 import scala.collection.JavaConverters._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/sink/ReThinkSinkConnector.scala:21]: and here

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceConnector.scala [hunk 9]
```diff
@@ -18,13 +18,6 @@ package com.datamountaineer.streamreactor.connect.rethink.source
...
 import java.util
 
-import com.datamountaineer.streamreactor.connect.rethink.config.{ReThinkSourceConfig, ReThinkSourceConfigConstants}
-import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.kafka.common.config.ConfigDef
-import org.apache.kafka.connect.connector.Task
-import org.apache.kafka.connect.source.SourceConnector
-import org.apache.kafka.connect.util.ConnectorUtils
-
 import scala.collection.JavaConversions._
```
Comment (carolinaloureiro) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceConnector.scala:21]: and here

**Review by carolinaloureiro — DISMISSED** [review]
wildcards

**Review by carolinaloureiro — APPROVED** [review]

**Review by goncaloccastro — APPROVED** [review]

File: kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceReader.scala [hunk 10]
```diff
@@ -56,15 +46,15 @@ object ReThinkSourceReader {
   }
 }
 
-class ReThinkSourceReader(rethink : RethinkDB, conn : Connection,
+class ReThinkSourceReader(rethink: RethinkDB, conn: Connection,
```
Comment (joseluisvf) [inline kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/source/ReThinkSourceReader.scala]: Missed this one :)

**Review by joseluisvf — CHANGES_REQUESTED** [review]

**Review by joseluisvf — APPROVED** [review]

**Pull request merged**